guardrail-lab/03-Mitigate/examples/hallucination-groundedness.md

```diff
@@ -0,0 +1,141 @@
+# Guardrail example: Groundedness check
```

Comment:
Is this file linked somewhere? I couldn't find it linked in the original lab and just wondering how it fits in the lab.

Reply:
I found that the whole `examples` folder is linked from the readme.